packages/lodestar/package.json

```diff
@@ -42,6 +42,9 @@
   "dependencies": {
     "@chainsafe/bit-utils": "^0.1.2",
     "@chainsafe/bls-js": "^0.1.5",
+    "@chainsafe/eth2.0-config": "^0.1.0",
+    "@chainsafe/eth2.0-params": "^0.1.0",
+    "@chainsafe/eth2.0-types": "^0.1.0",
```

Comment:
this is just interface? should also be dev dependency

Reply:
see above comment, we will be exposing functions that take the types as input/output, which will go in our generated type declaration files, so I think this should stay a dependency.

Reply:
In lodestar? If someone is using lodestar directly it will put @chainsafe/eth2.0-params in devdependency by itseld, although I'm pretty sure tsc will extract needed interfaces anyway

Reply:
For if someone is using lodestar just for its eg: state transition functions. I guess it's kind of far-fetched at this point.